crates/webidl/src/idl_type.rs

```diff
@@ -433,8 +433,10 @@ impl<'a> IdlType<'a> {
             IdlType::UnsignedShort => Some(ident_ty(raw_ident("u16"))),
             IdlType::Long => Some(ident_ty(raw_ident("i32"))),
             IdlType::UnsignedLong => Some(ident_ty(raw_ident("u32"))),
-            IdlType::LongLong => Some(ident_ty(raw_ident("i64"))),
-            IdlType::UnsignedLongLong => Some(ident_ty(raw_ident("u64"))),
+            // Note that these map to `f64` because JS doesn't really have a
+            // native type for u64, so WebIDL surely just means a JS number
+            IdlType::LongLong => Some(ident_ty(raw_ident("f64"))),
+            IdlType::UnsignedLongLong => Some(ident_ty(raw_ident("f64"))),
```

Comment:
This is a pretty subtle update in this PR, but lots of dictionaries apparently have `long long` specified in them. It's unclear what WebIDL could mean *other* than `f64` for JS because 64-bit integers aren't possible, so these two now map to f64

Reply:
From the [spec](https://heycam.github.io/webidl/#idl-long-long):
>The long long type is a signed integer type that has values in the range [−9223372036854775808, 9223372036854775807].
In other words, WebIDL says a long long is a 64-bit integer. Presumably with host-bindings this won't be an issue, it is however an issue with current day shims. I don't think we should change all the signatures of methods just because of modern day limitations.